.github/dependabot.yml

```diff
@@ -5,3 +5,18 @@ updates:
     directory: "/server"
     schedule:
       interval: "daily"
+  # yarn
+  - package-ecosystem: "npm"
+    directory: "/src"
```

Comment:
I think this should be where the `package.json` and `yarn.lock` are stored, so it should be `directory: "/"`